src/devtools/client/webconsole/components/Search/ConsoleSearch.tsx

```diff
@@ -48,18 +48,32 @@ export default function ConsoleSearch({}: Props) {
     actions.search(newQuery);
   };
 
+  let summaryMsg = null;
+  if (query) {
+    if (results.length > 0) {
+      if (results.length === 1) {
+        summaryMsg = "1 result";
+      } else {
+        summaryMsg = `${index + 1} of ${results.length} results`;
+      }
+    } else {
+      summaryMsg = "No results found";
+    }
+  }
+
```

Comment:
Going to extract this change into its own PR (#6544) since it's only tangentially test related.